### PR TITLE
Added SNES symbol

### PR DESCRIPTION
**Pull request by absindx**
I created a symbol definition file for the SNES I/O registers.
Register names and comments are based on the Super Nintendo Entertainment System Development Manual.

Thank you for the amazing software!

### CODE REVIEW

File: SourceGen/RuntimeData/SystemDefs.json [hunk 1]
```diff
@@ -263,6 +263,7 @@
       "Speed" : "3.58",
       "Description" : "Super Nintendo Entertainment System [placeholder].",
```
Comment (fadden) [inline SourceGen/RuntimeData/SystemDefs.json:264]: "[placeholder]" can probably be removed.

**Review by fadden — CHANGES_REQUESTED** [review]
Very exciting!  I think we need to review the MULT_MASK lines though.  Are the SNES registers repeated in memory?  I poked around a little and it looks like they're mirrored in banks $00-3f?

**Review by fadden — APPROVED** [review]
LGTM.  I know very little about SNES programming, but if it's appropriate you may want to add `"entry-flags":"native-long"` to the SystemDefs entry so that the processor flags default to 16-bit registers in native mode at entry points, instead of 8-bit emulation mode (see the second Apple IIgs entry for an example... sets the "Entry Flags" in General project properties).

**Comment by absindx** [comment]
Wow! I didn't know its function.
I set E = 0 every time from the project property settings.
In most cases, the flag will not change once switch to native mode on reset.
Thank you for the information!

**Comment by fadden** [comment]
Thanks for the contribution!  I can generate a "-dev" release if you want to share this.  I expect to do a stable minor release some time next week.

**Comment by absindx** [comment]
Thank you for your prompt and courteous response!
(SNES has multiple [memory mapping](https://en.wikibooks.org/wiki/Super_NES_Programming/SNES_memory_map) schemes and I'm a little unsure of how the bank works... I will contact you again if I find a problem.)
I'm looking forward to next week!

**Pull request merged**